extensions/amp-subscriptions/0.1/renderer.js

```diff
@@ -65,6 +71,17 @@ export class Renderer {
     });
   }
 
+  addLoadingBar() {
+    if (!this.getRootElement_().querySelector('[subscriptions-section=loading]')) {
+      const element = createElementWithAttributes(this.document_, 'div' , {
+        class: 'subscriptions-progress',
+        'subscriptions-section': 'loading',
+      });
+
+      this.getRootElement_().appendChild(element);
```

Comment:
`this.ampdoc_.getBody().appendChild()`. otherwise you'll add it to the `<html>` element. Unless that's what you were trying to do :)

Reply:
Done